### ISSUE TITLE
> 🔒Your funds are safe, once the transaction is confirmed, you will receive + 122346,00 NOT

### ISSUE THREAD

**Issue by ghost** [ton-blockchain/wallet-contract#33]
> 🔒Your funds are safe, once the transaction is confirmed, you will receive + 122346,00 NOT
> 
> _Originally posted by @wlofman021 in https://github.com/tonkeeper/opentonapi/issues/389_
> 
> _Originally posted by @wlofman021 in https://github.com/ton-blockchain/verifier/issues/215_

_Originally posted by @Ayosaja1234 in https://github.com/ton-blockchain/verifier/issues/216#issuecomment-2291326971_

**Comment by Ayosaja1234** [comment]
UQDO5GaJbGEfqIuQcrmhw-1rnB0Om8JOAx8KQjD_y5ImKyk3

**Comment by Ayosaja1234** [comment]
a44c821f51fc5622a4fc760e4fc648e8fe630b18

**Comment by JDJD24-af** [comment]
> UQDO5GaJbGEfqIuQcrmhw-1rnB0Om8JOAx8KQjD_y5ImKyk3

**Comment by belaya12** [comment]
Please  back my ton

**Comment by Behzadnasr** [comment]
🔒Receive: 100 TON + 0.24 TON

**Comment by Behzadnasr** [comment]
چرا این مقدارتون به ولت من واریز نشده

**Comment by Sofo4kadeto4ka** [comment]
Тон топ тон тон тон

**Comment by Kurdlove** [comment]
Hahaha

**Comment by belaya12** [comment]
Please back my dogs crypto

**Comment by MORTEZAMIRSALI** [comment]
Hello, don't be bored, everyone, Tonkeeper, you don't intend to cooperate, because I haven't been there for several years because of problems, instead of thanking my financial resources for helping to climb the tanks, all of them have created obstacles.  Be trustworthy people, he is trying to create an obstacle to own my belongings. Friends, I have been here for some time. Tunkeeper has not only spared 1 cent from my wealth, but he has reduced all the transactions from me on AirDrabs, but all of them are a violation of this example of violation and anti-citizen rights and abuse.  Absolutely from a citizen, if these were realistic and justice-oriented, even if my children came to control my assets for 60 years, there would not have been so much mistreatment and tension between all subscribers.  Don't stop trusting and be convinced of your right

**Comment by MORTEZAMIRSALI** [comment]
Tonkeeper, I'm an abuser, you know, I'm not a person who wants to interfere with other people's property. I've never been hurt by a single element.  The reason for your abuse will burn in this fire.  A phrase was added to return and the chance of the tank is very low, so you better understand and finish the robbery

**Comment by alisam1189** [comment]
> UQDO5GaJbGEfqIuQcrmhw-1rnB0Om8JOAx8KQjD_y5ImKyk3

Ok

**Comment by MORTEZAMIRSALI** [comment]
Of course, I respect you. During this time, the tonkeeper has accumulated my property in the tanks, and of course it has led to progress, but that is not the reason for this. After a while, the owner of the property will come and the tonkeeper will tell you where you were, why did you not receive your money.  There is a currency code, as you can see from its name. What do you care about me? Why didn't I take my money? I am really sorry.

**Comment by MORTEZAMIRSALI** [comment]
I remembered Trump's answer to the Tonkeeper's question. Most of his words were really honest. In such cases, he would say, "You fool, I'm sorry that I didn't take my money and I caused you to progress. Such managers don't know what a currency means. What does a code mean? Anonymous means secret. Then he says, where have you been?"  Wherever I was, wherever I was, do your job. Respecting someone has made you an idiot, and don't block him. In short, he forced himself to destroy me because he doesn't know what to do, and Trump's words should be covered with gold.
